src/shoobx/wfmc/process.py

```diff
@@ -1227,7 +1246,10 @@ def evaluate(self, expr, locals={}):
         ns.update(vars(self.process.workflowRelevantData))
         ns.update(vars(self.process.applicationRelevantData))
         ns.update(locals)
-        return eval(expr, ALLOWED_BUILTINS, ns)
+        try:
+            return eval(expr, ALLOWED_BUILTINS, ns)
+        except EVAL_EXCEPTIONS as e:
+            raise interfaces.EvaluateException from e
```

Comment:
Adding failed expression to EvaluateException would help with debugging.
```suggestion
            raise interfaces.EvaluateException(expr) from e
```